x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,352 @@
+                      <EuiSpacer size="s" />
+                    </EuiDescriptionListDescription>
+                  </EuiDescriptionList>
+                </EuiFlexItem>
+                {module.query !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexEnd">
+                      <EuiFlexItem grow={false}>
+                        <EuiText size={'xs'}>
+                          <FormattedMessage
+                            id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dataRecognizerHelpButtonLabel"
+                            defaultMessage="Check indices to see which can be used to run jobs in this module?"
+                          />
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <span>
+                          <EuiButton
+                            fill
+                            isDisabled={runningDataRecognizer}
+                            isLoading={runningDataRecognizer}
+                            color={'primary'}
+                            onClick={runDataRecongizer}
+                            size="s"
+                          >
+                            <FormattedMessage
+                              id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.runDataRecognizerButtonLabel"
+                              defaultMessage="Run data recognizer"
+                            />
+                          </EuiButton>
+                        </span>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                ) : null}
+                {matchingDataViews.length ? (
+                  <EuiFlexItem grow={false}>
+                    <DataViewsTable matchingDataViews={matchingDataViews} moduleId={module.id} />
+                  </EuiFlexItem>
+                ) : null}
+              </EuiFlexGroup>
+            ) : (
+              <EuiEmptyPrompt
+                css={{ margin: '10px' }}
+                color="primary"
+                iconType={(isLogoObject(module.logo) ? module.logo?.icon : module.logo) as string}
+                title={
+                  <h2>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleTitle"
+                      defaultMessage="You cannot create these jobs here"
+                    />
+                  </h2>
+                }
+                body={
+                  <p>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleHelpMessage"
+                      defaultMessage="These supplied configurations can be used in {appName}."
+                      values={{ appName: module.type }}
```

Comment:
Looks like we need to update the `type` field in the manifest files for the two metrics modules to replace 'Metricbeat data' with 'Infrastructure'.

Reply:
Updated type in f9a9bbc428dd0f41be050b0c3b22c63d69cea6f3

Reply:
This looks good. I don't suppose there are any locators for the Logs or Infrastructure pages so we could add in links?